core/src/main/kotlin/org/evomaster/core/output/service/GraphQLTestCaseWriter.kt

```diff
@@ -123,6 +125,19 @@ class GraphQLTestCaseWriter : HttpWsTestCaseWriter() {
             lines.append("${path?.let { GeneUtils.applyEscapes(it, mode = GeneUtils.EscapeMode.NONE, format = format) }}\"")
         }
 
+        if (format.isPython()) {
```

Comment:
isn't this the same code as done in `RestTestCaseWriter`? if so, should be refactored in a protected utility method, moved up in the hierarchy, eg in `HttpWsTestCaseWriter`